x-pack/plugins/canvas/canvas_plugin_src/strings/i18n.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable */
+
+import { i18n as i18nCore } from '@kbn/i18n';
+
+export const i18n =
```

Comment:
Ternaries are only for simple conditions, [nested ternaries are no good](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#only-use-ternary-operators-for-small-simple-code).